### PR TITLE
[FLINK-7181] Activate checkstyle flink-java/operators/*

### PR DESCRIPTION
**Pull request by dawidwys**
Thanks for contributing to Apache Flink. Before you open your pull request, please take the following check list into consideration.
If your changes take all of the items into account, feel free to open your pull request. For more information and/or questions please refer to the [How To Contribute guide](http://flink.apache.org/how-to-contribute.html).
In addition to going through the list, please provide a meaningful description of your changes.

- [ ] General
  - The pull request references the related JIRA issue ("[FLINK-XXX] Jira title text")
  - The pull request addresses only one issue
  - Each commit in the PR has a meaningful commit message (including the JIRA id)

- [ ] Documentation
  - Documentation has been added for new functionality
  - Old documentation affected by the pull request has been updated
  - JavaDoc for public methods has been added

- [ ] Tests & Build
  - Functionality added by the pull request is covered by tests
  - `mvn clean verify` has been executed successfully locally or a Travis build has passed

### CODE REVIEW

File: flink-java/src/main/java/org/apache/flink/api/java/operators/CoGroupRawOperator.java [hunk 1]
```diff
@@ -95,9 +96,8 @@ public CoGroupRawOperator(DataSet<I1> input1, DataSet<I2> input2,
 			int[] logicalKeyPositions1 = keys1.computeLogicalKeyPositions();
 			int[] logicalKeyPositions2 = keys2.computeLogicalKeyPositions();
 
-			CoGroupRawOperatorBase<I1, I2, OUT, CoGroupFunction<I1, I2, OUT>> po
-					= new CoGroupRawOperatorBase<I1, I2, OUT, CoGroupFunction<I1, I2, OUT>>(
-							function, new BinaryOperatorInformation<I1, I2, OUT>(getInput1Type(), getInput2Type(), 
+			CoGroupRawOperatorBase<I1, I2, OUT, CoGroupFunction<I1, I2, OUT>> po = new CoGroupRawOperatorBase<I1, I2, OUT, CoGroupFunction<I1, I2, OUT>>(
```
Comment (NicoK) [inline flink-java/src/main/java/org/apache/flink/api/java/operators/CoGroupRawOperator.java]: now this line is a bit too long (even if the caret shortcut `<>` was used)

**Comment by zentol** [comment]
+1 to merge.

**Comment by dawidwys** [comment]
Will rebase it and remove appropriate suppressions first.

**Comment by dawidwys** [comment]
merging

**Pull request closed without merging**